src/jdk.crypto.cryptoki/share/native/libj2pkcs11/pkcs11t.h

```diff
@@ -2173,6 +2179,16 @@ typedef struct CK_TLS12_MASTER_KEY_DERIVE_PARAMS {
 typedef CK_TLS12_MASTER_KEY_DERIVE_PARAMS CK_PTR \
         CK_TLS12_MASTER_KEY_DERIVE_PARAMS_PTR;
 
+typedef struct CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS {
+    CK_MECHANISM_TYPE prfHashMechanism;
+    CK_BYTE_PTR pSessionHash;
+    CK_ULONG ulSessionHashLen;
+    CK_VERSION_PTR pVersion;
```

Comment:
Is the order of fields significant? If not, why not use an order of a similar structure.
Also, please line up the field names like other structures declared in this file.

Reply:
This struct `CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS`, I copied it from the NSS library code, and followed the same order as NSS library code. The other structs defined in this pkcs11t.h file should also copied and followed the same order from the NSS library code.

I will line up the field names like other structures declared in this file.